cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -288,6 +290,127 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), "was not able to find the xt_u32 machine config in a machine config pool")
 		})
 	})
+	Context("OpenNESS Operator for Intel FPGA PAC N3000 (Programming)", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			err := testclient.Client.List(context.Background(), n3000NodeList)
+			if err != nil && err.Error() == "no matches for kind \"N3000Node\" in version \"fpga.intel.com/v1\"" {
+				Skip("Cluster doesn't have intel fpga pac n3000 card")
+			}
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should have installed OpenNESS Operator for Intel FPGA PAC N3000 (Programming) operator", func() {
+			Expect(len(n3000NodeList.Items) > 0).To(Equal(true), "Intel FPGA PAC N3000 (Programming) operator is not installed")
+		})
+
+		It("should expose the fortville nics on the intel fpga pac N3000 as part of the n3000Node CR", func() {
```

Comment:
Lets remove this.

The PR should only validate the operator deployment and this object will not exist on a cluster that doesn't have an N3000 card

this should be created on the e2esuite